schema/traces/traces.mapping

```diff
@@ -0,0 +1,220 @@
+{
+  "index_patterns": [
+    "sso_traces-*-*"
+  ],
+  "template": {
+    "mappings": {
+      "_meta": {
+        "version": "0.1.0-dev"
+      },
+      "dynamic_templates": [
+        {
+          "attributes_map": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "path_match": "attributes.*"
+          }
+        },
+        {
+          "events_attributes_map": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "path_match": "events.attributes.*"
+          }
+        },
+        {
+          "links_attributes_map": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "path_match": "links.attributes.*"
+          }
+        },
+        {
+          "instrumentation_scope_attributes_map": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "path_match": "instrumentationScope.attributes.*"
+          }
+        },
+        {
+          "resources_map": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "path_match": "resource.*"
+          }
+        }
+      ],
+      "_source": {
+        "enabled": true
+      },
+      "properties": {
+        "traceId": {
+          "ignore_above": 256,
+          "type": "keyword"
+        },
+        "spanId": {
+          "ignore_above": 256,
+          "type": "keyword"
+        },
+        "traceState": {
```

Comment:
This doesn't look right. Should `traceState` be object or nested?

README.md states `traceState` is a key-value pairs, `traces.schema` lists them as object.

Reply:
i'll update the schema doc